pydir/wcsfit.cpp

```diff
@@ -253,11 +256,17 @@ PYBIND11_MODULE(wcsfit, m) {
                     result["starDOF"] = vectorToNumpy(starCat.starDOF);
                     result["starChisq"] = vectorToNumpy(starCat.starChisq);
                     result["starX"] = vectorToNumpy(starCat.starX);
+                    result["ra"] = vectorToNumpy(starCat.starX);
```

Comment:
I've changed everywhere these are used in the stack, but I wanted to keep the old names because we have external collaborators using these data products, and I didn't want to break their analyses. It also preserves the original `gbdes` output format. Not 100% committed to this though.

Reply:
Could always make a new ticket later to remove the old names, I suppose.